SingularityService/src/main/java/com/hubspot/singularity/scheduler/SingularityUsageHelper.java

```diff
@@ -45,4 +103,472 @@ public SingularityUsageHelper(SlaveManager slaveManager) {
     return slavesToTrack;
   }
 
+  public Optional<SingularitySlaveUsage> collectSlaveUsageSimple(
+      SingularitySlave slave,
+      long now,
+      boolean useShortTimeout) {
+    Optional<Long> memoryMbTotal = Optional.absent();
+    Optional<Double> cpusTotal = Optional.absent();
+    Optional<Long> diskMbTotal = Optional.absent();
+
+    try {
+      List<MesosTaskMonitorObject> allTaskUsage = mesosClient.getSlaveResourceUsage(slave.getHost(), useShortTimeout);
+      MesosSlaveMetricsSnapshotObject slaveMetricsSnapshot = mesosClient.getSlaveMetricsSnapshot(slave.getHost());
+
+      double memoryMbReservedOnSlave = 0L;
+      double cpuReservedOnSlave = 0.0;
+      double diskMbReservedOnSlave = 0L;
+
+      double memoryBytesUsedOnSlave = 0;
+      double cpusUsedOnSlave = 0;
+      double diskMbUsedOnSlave = 0;
+
+      double systemMemTotalBytes = 0;
+      double systemMemFreeBytes = 0;
+      double systemLoad1Min = 0;
+      double systemLoad5Min = 0;
+      double systemLoad15Min = 0;
+      double slaveDiskUsed = 0;
+      double slaveDiskTotal = 0;
+      double systemCpusTotal = 0;
+      if (slaveMetricsSnapshot != null) {
+        memoryMbReservedOnSlave = slaveMetricsSnapshot.getSlaveMemUsed();
+        cpuReservedOnSlave = slaveMetricsSnapshot.getSlaveCpusUsed();
+        diskMbReservedOnSlave = slaveMetricsSnapshot.getSlaveDiskUsed();
+        systemMemTotalBytes = slaveMetricsSnapshot.getSystemMemTotalBytes();
+        systemMemFreeBytes = slaveMetricsSnapshot.getSystemMemFreeBytes();
+        systemLoad1Min = slaveMetricsSnapshot.getSystemLoad1Min();
+        systemLoad5Min = slaveMetricsSnapshot.getSystemLoad5Min();
+        systemLoad15Min = slaveMetricsSnapshot.getSystemLoad15Min();
+        slaveDiskUsed = slaveMetricsSnapshot.getSlaveDiskUsed();
+        slaveDiskTotal = slaveMetricsSnapshot.getSlaveDiskTotal();
+        systemCpusTotal = slaveMetricsSnapshot.getSystemCpusTotal();
+      }
+
+      for (MesosTaskMonitorObject taskUsage : allTaskUsage) {
+        String taskId = taskUsage.getSource();
+        SingularityTaskId task;
+        try {
+          task = SingularityTaskId.valueOf(taskId);
+        } catch (InvalidSingularityTaskIdException e) {
+          LOG.error("Couldn't get SingularityTaskId for {}", taskUsage);
+          continue;
+        }
+
+        SingularityTaskUsage latestUsage = getUsage(taskUsage);
+        memoryBytesUsedOnSlave += latestUsage.getMemoryTotalBytes();
```

Comment:
Will these end up being any different than `systemMemTotalBytes - systemMemFreeBytes` ? I'm wondering if there isn't actually a need to total everything up for these cases. `getUsage` is still a zk call and it'd be nice to eliminate if we can. The totaled up by task values and the slave-reported totals seem fairly similar. I'm not certain why we have both in the POJO TBH, would have to look through commit history. (Or maybe @darcatron knows since he wrote the original versions of usage collection?)